modules/manually-gathering-logs-with-SSH.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// *installing/installing-gather-logs.adoc
+
+[id="installation-manually-gathering-logs-with-SSH_{context}"]
+= Manually gathering logs with SSH access to your host(s)
+
+.Prerequisites
+
+* You must have SSH access to your host(s).
+
+.Procedure
+
+. Collect the `bootkube.service` service logs from the bootstrap host using the
+`journalctl` command by running:
++
+----
+$ journalctl -b -f -u bootkube.service
+----
+
+. Collect the bootstrap host's container logs using the podman logs. This is shown
```

Comment:
Init cap "Podman" if referring to the software name[1]? Or markup if referring to the tool specifically. I think it would be the former would be appropriate here, though.

[1] https://podman.io/